airflow/providers/weaviate/hooks/weaviate.py

```diff
@@ -675,7 +675,7 @@ def object_exists(self, uuid: str | UUID, **kwargs) -> bool:
 
     def _delete_objects(self, uuids: Collection, class_name: str, retry_attempts_per_object: int = 5):
         """
-        Helper function for `create_or_replace_objects()` to delete multiple objects.
+        Act as a helper function for `create_or_replace_objects()` to delete multiple objects.
```

Comment:
This can probably be more descriprive to what this function actually does. The helper function aspect can be put in the second line instead.

Reply:
Thanks. I will do that. I also did this in few other places. I'll take care of this issue there as well.